requirements.txt

```diff
@@ -14,13 +14,16 @@ black
 
 # Tests
 tox
-pytest
+pytest~=7.1.2
 pytest-cov
 pytest-asyncio; python_version >= '3.5'
 mock
-freezegun
+freezegun~=1.2.2
 
 # Typing
 types-mock
 types-freezegun
 types-redis
+
+pydantic~=1.10.7
```

Comment:
These should go to the extras, similar to the `redis` and `zstd` extras/dependencies: https://github.com/kpn/cachetory/blob/main/pyproject.toml#L37-L42

Reply:
updated